pkg/bosh/manifest/dns.go

```diff
@@ -207,12 +237,43 @@ func (dns *boshDomainNameService) Reconcile(ctx context.Context, namespace strin
 	return nil
 }
 
+func (dns *boshDomainNameService) createCorefile(namespace string) string {
+	rewrites := make([]string, 0)
+	for _, alias := range dns.Aliases {
+		for _, target := range alias.Targets {
+			from := alias.Domain
+			if target.Query == "_" {
+				from = strings.Replace(from, "_", target.InstanceGroup, 1)
+			}
+			to := fmt.Sprintf("%s.%s.svc.cluster.local", dns.HeadlessServiceName(target.InstanceGroup), namespace)
+			rewrites = append(rewrites, fmt.Sprintf("rewrite name exact %s %s", from, to))
+		}
+	}
+
+	tmpl := template.Must(template.New("Corefile").Parse(`
+.:8053 {
+    errors
+    health
+    {{- range $_, $rewrite := .Rewrites }}
+    {{ $rewrite }}
+    {{- end }}
+    forward . /etc/resolv.conf
+    cache 30
+    loop
+    reload
+    loadbalance
```

Comment:
Should we document why we chose these values?

Reply:
Fixed in https://github.com/cloudfoundry-incubator/cf-operator/pull/642/commits/fb0ce8852ec24a79116cc76359c720ba6e359b4b.